### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dimpsey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,103 @@
 # terraform-aws-ecs-scheduled-task
+
+A module to supports the ability to schedule tasks on either a cron -like schedule or in a response to CloudWatch Events in ECS.
```
Comment (yoonlees) [inline README.md]: We are not actually do: 
"in a response to CloudWatch Events in ECS".

Probably delete the words?

File: README.md [hunk 2]
```diff
@@ -1 +1,103 @@
...
+
+}
+```
+
+Argument Reference
+---------------------
+
+### Required
+
+* `name` -  The name of task to be scheduled.
```
Comment (yoonlees) [inline README.md]: Reword to indicate that name should be unique.

File: output.tf [hunk 3]
```diff
@@ -0,0 +1,7 @@
+output "task_definition_arn" {
+  value = "${local.task_definition_arn}"
+}
+
+output "subnet_ids" {
+  value = "${local.all_subnets}"
+}
```
Comment (yoonlees) [inline output.tf]: Want to add more to the outputs like schedule rule name and its target name?

File: locals.tf [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# task_defintion_arn
+locals {
+  task_definition_arn = "${element(concat(aws_ecs_task_definition.fargate.*.arn,
```
Comment (yoonlees) [inline locals.tf]: task_definition_arns better?

File: README.md (hunk 1 above)
Comment (dimpsey) [reply README.md]: done.

**Pull request merged**